README.md

```diff
@@ -4,8 +4,28 @@
 [![codecov](https://codecov.io/gh/vicariousinc/PGMax/branch/master/graph/badge.svg?token=FrRlTDCFjk)](https://codecov.io/gh/vicariousinc/PGMax)
 [![Documentation Status](https://readthedocs.org/projects/pgmax/badge/?version=latest)](https://pgmax.readthedocs.io/en/latest/?badge=latest)
 
+# Important note: PGMax is now part of DeepMind 🎉!
+
+<body>
+<h1 align="center">
+Following the acquisition of <a href="https://intrinsic.ai/blog/posts/mission-momentum">Vicarious by Alphabet</a> in 2022, we are excited to share that we have now ported over PGMax to <a href="https://github.com/deepmind/PGMax">DeepMind's GitHub</a>! 
+<br>
+<br>
+Our integration within DeepMind's rapidly growing JAX ecosystem opens up new promising development for PGMax and new exciting research possibilities.
+<br>
+<br>
+Consequently, this repository will not be updated in the future, and all the new updates will be pushed to
+<a href="https://github.com/deepmind/PGMax">DeepMind's repository</a>.
+</h1>
+</body>
+<br>
+<br>
+
+
 # PGMax
 
+## Description
```

Comment:
We should also update the descriptions and get rid of the obsolete info to avoid confusing people.

One option is just remove all the old descriptions.

Another is to slightly update the old descriptions. Things I can think of:

1. Remove blog post link
2. Update citation entry
3. Maybe update installation instructions

Reply:
good point: let's just remove all the descriptions